.appveyor.yml

```diff
@@ -1,5 +1,5 @@
 environment:
-  ARMADILLO_DOWNLOAD: "https://sourceforge.net/projects/arma/files/armadillo-9.800.6.tar.xz"
+  ARMADILLO_DOWNLOAD: "http://ftp.fau.de/macports/distfiles/armadillo/armadillo-8.400.0.tar.xz"
```

Comment:
@zoq is that version downgrade deliberate?
recent releases of ensmallen require armadillo 9.800+